### PR TITLE
Material Sample documentation

### PR DESCRIPTION
**Pull request by ljwalker**
Now includes traits, material sample with updated links and screenshot

### CODE REVIEW

File: content/editor/edit/tabs/_index.en.md [hunk 1]
```diff
@@ -16,12 +16,17 @@ Upon opening the Record Editor page, you will see 4-5 tabs, depending on your le
...
 
-Determination History – where annotations are shown and can be entered. Annotations can also be batch uploaded from separate files. See [Annotations](https://biokic.github.io/symbiota-docs/editor/edit/annotations/)
+**Determination History** – where annotations are shown and can be entered. Annotations can also be batch uploaded from separate files. See [Annotations](https://biokic.github.io/symbiota-docs/editor/edit/annotations/)
 
-Images – provides information on image(s) associated with the record. See [Images](https://biokic.github.io/symbiota-docs/editor/images/)
+**Images** – provides information on image(s) associated with the record. See [Images](https://biokic.github.io/symbiota-docs/editor/images/)
 
-Linked Resources – allows linkage of checklists, duplicates, and genetic resources to the record
+**Material Sample** - allows for association of multiple material sample records per specimen occurrence. For this tab to be visible, it must be configured per portal and activated per collection. Contact your Collection Administrator for more information. See [Linking Records/Resources](https://biokic.github.io/symbiota-docs/editor/materialsample/)
```
Comment (themerekat) [inline content/editor/edit/tabs/_index.en.md]: Should the text within the brackets be "Material Sample Info" or something like that?

Comment (themerekat) [inline content/editor/edit/tabs/_index.en.md]: Should the text within the brackets be "Material Sample Info" or something like that?

**Review by themerekat — CHANGES_REQUESTED** [review]

File: content/editor/edit/tabs/_index.en.md (hunk 1 above)
Comment (ljwalker) [reply content/editor/edit/tabs/_index.en.md]: Fixed now, I think.

**Comment by ljwalker** [comment]
Latest commits include an entirely new page for the Editor guide on Material Sample, as well as additions to the Symbiota Data Fields page to include Material Sample.

**Review by themerekat — APPROVED** [review]

**Pull request merged**